ngx-fudis/projects/ngx-fudis/src/lib/components/button/button.stories.ts

```diff
@@ -1,16 +1,20 @@
 import { StoryFn, Meta } from '@storybook/angular';
 import { ButtonComponent } from './button.component';
 import readme from './readme.mdx';
+import { buttonExclude } from '../../utilities/storybook';
 
 export default {
   title: 'Components/Button',
   component: ButtonComponent,
   parameters: {
     docs: {
+      argTypes: {
+        exclude: buttonExclude,
```

Comment:
For some reason (similar to Dialog) this didn't work and readme ArgTypes needed the exclude. So I deleted this one and added the exclude in readme.